app/scripts/Annotations1dTrack.js

```diff
@@ -53,10 +55,21 @@ class Annotations1dTrack extends PixiTrack {
     const graphics = this.pMain;
     graphics.clear();
 
+    const timeStamp = performance.now();
```

Comment:
How is this used?

Reply:
I though I replied but maybe I didn't.

This is used to figure out which rect graphics object are still visible and which ones can be destroyed. For every draw cycle a new time stamp is generated and all visible rects get this new time stamp. rect graphics without this current time stamp are then destroyed.